concreate/cli.py

```diff
@@ -45,6 +47,10 @@ def run(self):
         build_group = parser.add_argument_group(
             'build', "Arguments valid for the 'build' target")
 
+        build_group.add_argument('--build-engine',
+                                 default='docker',
+                                 help='An engine used to build an image [docker,osbs]')
```

Comment:
You should specify these using this: https://docs.python.org/3/library/argparse.html#choices